userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -150,6 +150,12 @@ existing cluster:
 $ eksctl create nodegroup --managed
 ```
 
+Tip : if you are using only one *`YOUR_CLUSTER.yaml`* file to describe your whole cluster, describe your new managed node group in its `managedNodeGroups` directive and run\:
+
+```console
+$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml --include='YOUR_MANAGEDNODEGROUP_NAME'
```

Comment:
By default, `eksctl create nodegroup` detects non-existing nodegroups and creates them, so `--include` can be omitted. 

```suggestion
$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml'
```